docs/config.toml

```diff
@@ -70,7 +70,7 @@ weight = 1
 #copyright = "Skaffold"
 #privacy_policy = "https://policies.google.com/privacy"
 github_repo = "https://github.com/GoogleContainerTools/skaffold"
-skaffold_version = "skaffold/v1beta4"
+skaffold_version = "skaffold/v1beta5"
```

Comment:
Should that be done on a new release rather than during the development of a new schema?
We had to split `examples` and `integration/examples` because of that. I'd love to find a better way though.

Reply:
well, the webpage doesn't gate deployed to skaffold.dev until a real release. For merges on master, you get the latest version deployed on https://skaffold-latest.firebaseapp.com. So I think this is fine!